leaf-ui/rappid-extensions/AnalysisInspector.js

```diff
@@ -459,228 +224,232 @@ var AnalysisInspector = Backbone.View.extend({
 				absTimeValues.push(aTime);
 			}
 		}
-	absTimeValues.sort();
-	if (absTimeValues[0] === ""){
-		absTimeValues.splice(0,1)
-	}
-	console.log(absTimeValues)
-	for (var s = 0; s < absTimeValues.length; s++) {
-		$('#header-row').append('<th>Absolute</th>');
-		$('#intentionRows').append('<th>' + absTimeValues[s] + '</th>');
-	}
-
-
-	//loop over intentions to get intial values and funcType
-	for (var i = 0; i < model.intentions.length; i++) {
-		var intention = model.intentions[i];
-		var initValue = intention.getInitialSatValue();
-		var func = intention.dynamicFunction.stringDynVis;
-
-		var row = $('<tr></tr>');
-		row.addClass('intention-row');
-		var name = $('<td></td>');
-		var sat = $('<td></td>');
-
-		name.text(intention.nodeName);
-		sat.text('Denied');
-		row.append(name);
-		row.append(satisfactionValuesDict[initValue].satValue);
-
-		for (j = 0; j < absTimeValues.length; j++) {
-			var options = ``;
-
-			// Add select tags for each absolute time point
-			var selectTd = $('<td></td>');
-			var selectElement = $('<select></select>');
-			selectElement.attr('nodeID', intention.nodeID);
-			selectElement.attr('absTime', absTimeValues[j]);
-
-
-			if (func === "I" || func === "D" || func === "C" || func === "R") {
-				switch (func) {
-					case "I":
-						options = this.increasing(initValue,'noFinal');
-						break;
-					case "D":
-						options = this.decreasing(initValue,'noFinal');
-						break;
-					case "C":
-						options = this.constant(initValue);
-						break;
-					case "R":
-						options = this.stochastic();
-						break;
-				}
-			}
-			else if (func === "MP" || func === "MN" || func === "CR" || func === "RC" || func === "SD" || func === "DS") {
-				//check for every node if there is assigned time
-				for (var k = 0; k < constraints.length; k++) {
-					if (constraints[k].constraintSrcID === intention.nodeID) {
-						var c = k
+		absTimeValues.sort(function(a,b) {return a-b});
+		if (absTimeValues[0] === ""){
+			absTimeValues.splice(0,1)
+		}
+		console.log("absTimeLength = " + absTimeValues.length);
+		for (var s = 0; s < absTimeValues.length; s++) {
+			$('#header-row').append('<th>Absolute</th>');
+			$('#intentionRows').append('<th>' + absTimeValues[s] + '</th>');
+		}
+
+
+		//loop over intentions to get intial values and funcType
+		console.log("model intentions length = " + model.intentions.length);
+		console.log("updated");
```

Comment:
Can this console log be more specific? What is being updated?

Reply:
I removed completely - was used for debugging and didn't have purpose anymore. Sorry I missed that in my first run throughs!